src/Plugin/Block/ActivityFinder4Block.php

```diff
@@ -175,6 +202,7 @@ public function build() {
         'mobile' => $image_mobile,
         'desktop' => $image_desktop,
       ],
+      '#bootstrap_version' => $bs_version,
```

Comment:
"bootstrap_version" VS "bs_version" - could you please change it to use "bs_version", "bs-version" and "bsVersion"?

Reply:
Fixed,